### PR TITLE
[EDU-4242] feat: add app modernization archi

### PR DESCRIPTION
**Pull request by gabriel-azion**

### CODE REVIEW

File: src/content/docs/en/pages/architectures/edge-application/application-modernization.mdx [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+title: Accelerate Application Modernization at the Edge
+description: Discover how Azion's Edge Platform enables application modernization through a serverless approach and microservices architecture. Learn how to migrate legacy applications to the edge, improving speed, reliability, and cost efficiency.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/edge-application/application-modernization.mdx:3]:
```suggestion
description: Discover how Azion Edge Platform enables application modernization through a serverless approach and microservices architecture.
```

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/architectures/edge-application/application-modernization.mdx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+4. Test and monitor your edge application.
+
+---
+
+## Related docs
+
+- [Azion Marketplace](/en/documentation/products/marketplace/).
+- [Azion Edge Runtime Web APIs compatibility](/en/documentation/products/devtools/azion-edge-runtime/frameworks-compatibility/).
+- [Azion Edge Runtime Node.js compatibility](/en/documentation/products/azion-edge-runtime/compatibility/node/).
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/edge-application/application-modernization.mdx:86]:
```suggestion
- [Azion Marketplace](/en/documentation/products/marketplace/)
- [Azion Edge Runtime Web APIs compatibility](/en/documentation/products/devtools/azion-edge-runtime/frameworks-compatibility/)
- [Azion Edge Runtime Node.js compatibility](/en/documentation/products/azion-edge-runtime/compatibility/node/)
```

File: src/content/docs/en/pages/architectures/edge-application/application-modernization.mdx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+4. The content, if static, will be stored in Edge Storage. So, it'll be requested and returned as a response.
+
+5. Operations requiring database transactions can be accomplished with Edge SQL.
+
+6. The application's logs are sent through Data Stream and can be analyzed by the customer.
+
+7. If it's necessary to access an origin outside Azion:
+    - Activating Origin Shield, your application is protected from a network list maintained by Azion.
+    - Activating the Load Balancer module balances traffic across your origins, data centers, or cloud providers and mitigates network congestion and server overload.
+    - Auth Origin (?)
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/edge-application/application-modernization.mdx:40]: Validate this item

File: src/content/docs/en/pages/architectures/edge-application/application-modernization.mdx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+- [Azion Marketplace](/en/documentation/products/marketplace/) is a digital catalog that makes it easy to find, test, and deploy edge-running software.
+- [Azion CLI](/en/documentation/products/azion-cli/overview/):  an open-source command-line interface (CLI) that lets you interact with Azion Edge Platform using a terminal. 
+- [Azion Console](/en/documentation/products/guides/getting-to-know-azion-console/): Azion’s configuration interface where you can access all the available products and their settings. It enables you to complete all tasks in Azion’s platform
+
+---
+
+## Implementation
+
+1. Create an edge application:
+    - On [Console](https://console.azion.com), create an application using the + Create button on the homepage.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/edge-application/application-modernization.mdx:65]:
```suggestion
    - On [Console](https://console.azion.com), create an application using the **+ Create** button on the homepage.
```

File: src/content/docs/en/pages/architectures/edge-application/application-modernization.mdx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+2. Create a domain and associate it with the edge application: 
+    - On Console, [create a domain](/en/documentation/products/guides/configure-a-domain/) using the + Create button on the homepage.
+    - Use the [Azion API](https://api.azion.com/#d43c5509-fb99-4b20-a7bd-64f0924af352) to create a domain.
+    - If you created an application through a template, the domain is created for you.
+3. Point domain to Azion: [update your DNS settings](/en/documentation/products/guides/migrate-ns-to-azion/) to point your domain's CNAME record to the Azion CDN endpoint provided for your domain. This step isn't mandatory.
+
+:::note 
+The edge application is accessible through the domain provided by Azion, which looks similar to: `ftd1cvntfl.map.azionedge.net`. 
+:::
+
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/architectures/edge-application/application-modernization.mdx:77]:
```suggestion
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+    - **Edge Cache**: módulo global para armazenar conteúdo no edge.
+    - **Edge Functions**: permite que você crie aplicações serverless orientadas a eventos, no edge, mais próximas dos usuários.
+    - **Rules Engine**: uma ferramenta para configurar os cenários em que uma política de cache específica é aplicada.
+    - **Tiered Cache**: módulo que cria uma camada de cache adicional entre o edge e seus servidores de origem, reduzindo a latência e os custos de infraestrutura.
+    - **Load Balancer**: módulo projetado para balancear o tráfego entre suas origens, data centers ou provedores de nuvem, mitigando a congestão da rede e a sobrecarga do servidor.
+    - **Edge Storage**: um serviço de armazenamento escalável e seguro projetado para integrar armazenamento de objetos com a Plataforma de Edge da Azion, usando o padrão S3 para operações de objetos.
+    - **Origns**: permite que você personalize o endereço de origem do conteúdo da sua edge application, o cabeçalho Host da sua aplicação, o caminho e as credenciais de autenticação HMAC para serviços protegidos.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx:53]:
```suggestion
    - **Origins**: permite que você personalize o endereço de origem do conteúdo da sua edge application, o cabeçalho Host da sua aplicação, o caminho e as credenciais de autenticação HMAC para serviços protegidos.
```

File: src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+permalink: /documentacao/arquiteturas/edge-application/modernizacao-de-aplicacoes/
+---
+
+Na Azion, a modernização de aplicações é um processo que atualiza aplicações legados para se alinharem com as demandas dinâmicas dos ambientes de negócios modernos. É implementado usando uma abordagem serverless e arquitetura de microsserviços em uma infraestrutura de edge, projetando seu sistema para simplicidade operacional, implantação mais rápida e escalabilidade.
+
+A Azion otimiza o desempenho processando dados mais próximos de sua fonte, reduzindo drasticamente a latência e melhorando a experiência do usuário. O modelo serverless reduz os custos ao eliminar a gestão de servidores e os fluxos de trabalho DevOps, facilitando a eficiência de negócios, competitividade e adoção de novas estratégias.
+
+Sua aplicação pode ser completamente hospedado na Azion, mas há também a possibilidade de ter apenas partes do sistema rodando no edge. Esta possibilidade ajuda na migração de aplicações legadas para aplicações modernas no edge, pois você pode executar a transição parte por parte, adaptando tudo e testando para garantir que atenda às suas necessidades.
```
Comment (GabrielAzion) [inline src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx:13]: hospedada

File: src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx [hunk 8]
```diff
@@ -0,0 +1,80 @@
+---
+título: Acelere a Modernização de Aplicações no Edge
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx:2]:
```suggestion
título: Acelere a modernização de aplicações no edge
```

File: src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+descrição: Descubra como a Azion possibilita a modernização de aplicações por meio de uma abordagem serverless e arquitetura de microsserviços.
+meta_tags: edge, aplicações, aplicações web, api, entrega de conteúdo, cdn, serverless, Azion
+namespace: docs_arch_app_modernization
+permalink: /documentacao/arquiteturas/edge-application/modernizacao-de-aplicacoes/
+---
+
+Na Azion, a modernização de aplicações é um processo que atualiza aplicações legados para se alinharem com as demandas dinâmicas dos ambientes de negócios modernos. É implementado usando uma abordagem serverless e arquitetura de microsserviços em uma infraestrutura de edge, projetando seu sistema para simplicidade operacional, implantação mais rápida e escalabilidade.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx:9]:
```suggestion
Na Azion, a modernização de aplicações é um processo que atualiza aplicações legadas para se alinharem com as demandas dinâmicas dos ambientes de negócios modernos. É implementado usando uma abordagem serverless e arquitetura de microsserviços em uma infraestrutura de edge, projetando seu sistema para simplicidade operacional, implantação mais rápida e escalabilidade.
```

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+
+## Arquitetura
+
+![Visão geral de uma requisição/resposta via Edge Application](/assets/docs/images/uploads/modernizacao-aplicação.png)
+
+### Fluxo de dados
+
+1. O usuário acessa o Domain relacionado a Edge Application que está sendo acessado. Essa requisição vai para o Roteador de Tráfego do Azion, que escolhe o melhor edge node para receber a requisição e direciona a requisição para este edge node.
+
+2. A requisição passa pelo Edge Firewall e é analisada. Se não for uma requisição maliciosa, ela chega a Edge Application.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx:25]:
```suggestion
2. A requisição passa pelo Edge Firewall e é analisada. Se não for uma requisição maliciosa, ela chega à Edge Application.
```

File: src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+
+3. A requisição chega a edge application e, com base em sua configuração do Rules Engine, acessa o conteúdo solicitado. Se configurados, os módulos de Image Processor, Edge Cache, Application Accelerator e Edge Functions serão executados.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx:27]:
```suggestion
3. A requisição chega à edge application e, com base em sua configuração do Rules Engine, acessa o conteúdo solicitado. Se configurados, os módulos de Image Processor, Edge Cache, Application Accelerator e Edge Functions serão executados.
```

File: src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+Com Edge Functions, você pode ter a lógica executada no edge, melhorando a usabilidade e manutenibilidade de aplicações descentralizadas. Essas funções são escritas em Javascript e executadas no Azion Edge Runtime.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx:28]:
```suggestion
Com Edge Functions, você pode ter a lógica executada no edge, melhorando a usabilidade e manutenibilidade de aplicações descentralizadas. Essas funções são escritas em Javascript e executadas no Edge Runtime da Azion.
```

File: src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx [hunk 13]
```diff
@@ -0,0 +1,80 @@
...
+    - **Edge Storage**: um serviço de armazenamento escalável e seguro projetado para integrar armazenamento de objetos com a Plataforma de Edge da Azion, usando o padrão S3 para operações de objetos.
+    - **Origins**: permite que você personalize o endereço de origem do conteúdo da sua edge application, o cabeçalho Host da sua aplicação, o caminho e as credenciais de autenticação HMAC para serviços protegidos.
+- **Domains**: registre um domínio personalizado com a Azion para entregar sua edge application.
+
+---
+
+## Implementação
+
+1. Crie um edge application:
+    - No [Console](https://console.azion.com), crie um aplicação usando o botão **+ Criar** na página inicial.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx:61]:
```suggestion
    - No [Console](https://console.azion.com), crie um aplicação usando o botão **+ Create** na página inicial.
```

File: src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx [hunk 14]
```diff
@@ -0,0 +1,80 @@
...
+1. Crie uma edge application:
+    - No [Console](https://console.azion.com), crie uma aplicação usando o botão **+ Create** na homepage.
```
[the remark this answers is not in the document]
Comment (MarianaAguilera) [reply src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx:63]: duplicado

File: src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx [hunk 15]
```diff
@@ -0,0 +1,80 @@
...
+    - Use a [Azion API](https://api.azion.com/#d43c5509-fb99-4b20-a7bd-64f0924af352) para criar uma edge application.
+    - Use [Azion CLI] para criar e implantar uma edge application.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx:65]:
```suggestion
    - Use [Azion CLI](/pt-br/documentacao/devtools/cli/create/#edge-applications) para criar e implantar uma edge application.
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx (hunk 7 above)
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/arquiteturas/edge-application/application-modernization.mdx:13]:
```suggestion
Sua aplicação pode ser completamente hospedada na Azion, mas há também a possibilidade de ter apenas partes do sistema rodando no edge. Esta possibilidade ajuda na migração de aplicações legadas para aplicações modernas no edge, pois você pode executar a transição parte por parte, adaptando tudo e testando para garantir que atenda às suas necessidades.
```

**Pull request merged**